dfc/tests/regression_test.go

```diff
@@ -260,38 +263,58 @@ func TestGetAndReRegisterInParallel(t *testing.T) {
 				}
 				_, _, err := client.Get(proxyurl, bucket, SmokeStr+"/"+repFile.filename, nil, nil, false, false)
 				if err != nil {
-					atomic.AddUint64(&numGetErrs, 1)
+					r := atomic.LoadUint64(&reregistered)
+					if r == 1 {
+						atomic.AddUint64(&numGetErrsAfter, 1)
+					} else {
+						atomic.AddUint64(&numGetErrsBefore, 1)
+					}
 				}
 				atomic.AddUint64(&getsCompleted, 1)
-				if (getsCompleted)%5000 == 0 {
-					tlogf(" %d/%d GET requests completed...\n", getsCompleted, num*numGetsForEachFile)
+				g := atomic.LoadUint64(&getsCompleted)
```

Comment:
nitpick: AddUint64 returns a new value, so you can combine two lines in one

Reply:
I used a variable here because the value is used twice below

Reply:
I meant this line and above, not below )